tests/input/test_input_gg_checks.py

```diff
@@ -0,0 +1,87 @@
+"""Test suite for parse_config function."""
+
+import pytest
+from gatorgrade.input.in_file_path import parse_config
+
+
+def test_parse_config_gg_check_in_file_context_contains_file():
+    """Test to make sure gatorgrader checks that have a file context include the file name in the list of lists."""
+    # Given the path to the test yml file
+    config = "tests/input/yml_test_files/gatorgrade_one_gg_check_in_file_context.yml"
+    # when the parse_config function is run
+    output = parse_config(config)
+    # then assert that the file path exists in the first index of the list
+    assert "file.py" in output["gatorgrader"][0]
+
+
+def test_parse_config_check_gg_matchfilefragment():
+    """Test to make sure keywords like MatchFileFragment appear inside the gator grader check list."""
+    # Given the path to the test yml file
+    config = "test/input/yml_test_files/gatorgrade_matchfilefragment.yml"  # Make fixture that will create yml file to be parsed instead of expected file
```

Comment:
If you don't plan on doing this before this pull request is merged, you should probably remove this comment and make it an issue in the issue tracker so that it can be seen more easily by those who work on it in the future.